lmdeploy/pytorch_poc/kernels/rms_norm.py

```diff
@@ -6,42 +6,94 @@
 
 
 @triton.jit
-def rms_norm_kernel(X, WEIGHT, OUT, eps, BLOCK_N: tl.constexpr):
+def rms_norm_kernel(input, weight, output, input_row_stride, n_cols, eps,
```

Comment:
n_cols is redundant of N_COLS

Reply:
`type(n_cols)` is tensor while `type(N_COLS)` is int, not same one.